avidemux.spec

```diff
@@ -184,7 +191,7 @@ rm -rf build_plugins_cli && mkdir build_plugins_cli && pushd build_plugins_cli
        -DUSE_EXTERNAL_LIBASS=TRUE \
        -DUSE_EXTERNAL_LIBMAD=TRUE \
        -DUSE_EXTERNAL_LIBA52=TRUE \
-       -DUSE_EXTERNAL_TWOLAME=TRUE \
+       -DUSE_EXTERNAL_MP4V2=TRUE \
        ../avidemux_plugins
```

Comment:
CMake variable USE_EXTERNAL_TWOLAME relied on an obsolete part of the former `avidemux-2.6.10-bundled_libs.patch` downstream patch. Avidemux doesn't use bundled twolame since end of May 2012: [[twolame] Use system twolame](https://github.com/mean00/avidemux2/commit/1e0c31e58226c8f1bf3f069a8f011f8293de8de6).